openlibrary/plugins/upstream/tests/test_merge_authors.py

```diff
@@ -68,7 +76,8 @@ def test_MockSite():
     ])
     assert site.docs.keys() == ["a", "b"]
```

Comment:
This will never be True in Python 3.
```suggestion
    assert list(site.docs) == ["a", "b"]
```

Reply:
Fixed in one of your PRs; rebased.